src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12350,6 +12350,42 @@ public void TestWarnAsErrorMinusDoesNotEnableDisabledByDefaultAnalyzers(Diagnost
             }
         }
 
+        [WorkItem(49446, "https://github.com/dotnet/roslyn/issues/49446")]
+        [CombinatorialData, Theory]
+        public void TestWarnAsErrorMinusDoesNotNullifyEditorConfig(bool warnAsErrorMinus)
```

Comment:
>TestWarnAsErrorMinusDoesNotNullifyEditorConfig [](start = 20, length = 46)

Given the nature of the change, I think we need to test more permutations of command line settings and config settings. I would say across the existing and new tests we should cover full matrix.